apps/posts/src/views/comments/components/comments-list.tsx

```
@@ -105,17 +99,16 @@ function CommentContent({item}: {item: Comment}) {
 
     return (
         <div className={`mt-1 flex flex-col gap-2`}>
-            <div className="flex max-w-[720px] flex-col items-start">
+            <div className={`flex max-w-[720px] flex-col items-start ${item.status === 'hidden' && 'opacity-50'}`}>
                 <div
                     dangerouslySetInnerHTML={{__html: item.html || ''}}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)